examples/hotspot.rs

```diff
@@ -0,0 +1,29 @@
+extern crate network_manager;
+
+use std::env;
+use std::process;
+
+use network_manager::manager;
+use network_manager::device;
+use network_manager::connection;
+
+
+fn main() {
+    let args: Vec<String> = env::args().collect();
+
+    if args.len() < 3 {
+        println!("USAGE: hotspot SSID PASSWORD");
```

Comment:
It would be nice if we could optionally pass in the wifi interface to be used, same with the password so the user can make an open hotspot